cmake/kconfig.cmake

```diff
@@ -104,6 +104,38 @@ if(EXTRA_KCONFIG_OPTIONS)
     )
 endif()
 
+# Set overlay config files list using a list of directories where the
+# overlay files could be found.
+if(DEFINED ENV{OVERLAY_CONFIG_DIRS})
+  set(OVERLAY_CONFIG_DIRS $ENV{OVERLAY_CONFIG_DIRS})
+
+  # We add current directory first so that it is seached first
+  set(overlay_config_dir_list ".:${OVERLAY_CONFIG_DIRS}")
+  string(REPLACE ":" ";" overlay_config_dirs "${overlay_config_dir_list}")
+
+  foreach(f IN ITEMS ${OVERLAY_CONFIG_AS_LIST})
+    if(IS_ABSOLUTE ${f})
+      list(APPEND overlay_config_files ${f})
+    else()
+      set(found 0)
+      foreach(d ${overlay_config_dirs})
+        if(EXISTS ${d}/${f} AND NOT IS_DIRECTORY ${d}/${f})
+          message(STATUS "Overlaying ${d}/${f}")
```

Comment:
Doesn't kconfig.py spit this out?

Reply:
I did not see it doing that.

Reply:
Loading /home/sebo/ncs/zephyr/boards/arm/nrf9160_pca10090/nrf9160_pca10090_defconfig as base
Merging /home/sebo/ncs/zephyr/samples/hello_world/prj.conf

Reply:
The system definitely does not print anything for the overlay config file atm.

Reply:
That is strange, can you find out why?

Perhaps it is being omitted.